modules/core/src/main/java/org/apache/ignite/internal/processors/cache/checker/processor/ReconciliationEventListenerFactory.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.checker.processor;
+
+/**
+ * Event listener factory for testing purposes.
+ */
+public class ReconciliationEventListenerFactory {
```

Comment:
It isn't factory at all. May be provider, but not factory because it doesn't produce anything.